database/nostr-lmdb/src/store/mod.rs

```diff
@@ -26,8 +25,22 @@ pub struct Store {
     ingester: Sender<IngesterItem>,
 }
 
+impl Clone for Store {
```

Comment:
I would remove this clone for now, since it's never used. Also `NostrLmdb` doesn't implement `Clone` so this is not needed currenlty. Eventually can be derived in a dedicated commit/PR